google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/PG_ClientSideStatements.json

```diff
@@ -133,6 +133,14 @@
       "method": "statementShowTransactionIsolationLevel",
       "exampleStatements": ["show transaction isolation level","show variable transaction isolation level"]
     },
+    {
+      "name": "EXPLAIN '<sql>'",
+      "executorName": "ClientSideStatementNoParamExecutor",
```

Comment:
This won't work, as [the method that you have defined](https://github.com/googleapis/java-spanner/blob/242e35142357ad55d27444437320b37d4107a560/google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java#L458) has a parameter; the sql string that is to be explained is the input parameter in this case. You have two options for this:
1. You could reuse the `ClientSideStatementSetExecutor` for this. That executor is normally used for statements like `SET <variable> = <value>` where the `value` is the input parameter. In this case, the input parameter would be the entire sql string that follows `EXPLAIN`.
2. You could create a custom executor specifically for this method. See `ClientSideStatementPgBeginExecutor` for an example for that.

Reply:
I think creating a new executor class will be better as ClientSideStatementSetExecutor is for specific type of commands and EXPLAIN clearly doesn't fit in that type. What do you say?

Reply:
Yes, that would be the neatest solution for this.